src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-new-customer-modal/index.js

```diff
@@ -86,37 +87,6 @@ export default {
             return this.customer !== null ? this.customer.addresses.get(this.customer.defaultBillingAddressId) : null;
         },
 
-        isSameBilling: {
```

Comment:
the problem regarding computed isSameBilling call twice leads to new address added twice

<img width="466" alt="Screenshot_2024-12-05_at_14 00 38" src="https://github.com/user-attachments/assets/b15a010b-ee5e-44ce-bcce-e8404821fb38" />